semantics/executable-spec/src/Data/MemoBytes.hs

```diff
@@ -87,6 +94,11 @@ printMemo x = putStrLn (showMemo x)
 memoBytes :: Encode w t -> MemoBytes t
 memoBytes t = Memo (runE t) (shorten (toLazyByteString (encode t)))
 
+memoBytesAnn :: Encode w (Annotator t) -> MemoBytes t
+memoBytesAnn t = Memo term (shorten bytes)
+   where bytes = toLazyByteString (encode t)
+         Annotator f = runE t
+         term = f (Full bytes)
```

Comment:
The `Annotator` type is produced when decoding. I would not expect it to ever appear in an encoder like this.

Since this appears to not be used anywhere at the moment, and locations where it is used are likely to have arrived at the need for it by mistake, I suggest removing it.

Reply:
Yes this should have been removed earlier.